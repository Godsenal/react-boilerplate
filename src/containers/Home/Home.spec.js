import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom'; // Test router component.
import { Home } from '.';

describe('Home snapshot test', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Home />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
