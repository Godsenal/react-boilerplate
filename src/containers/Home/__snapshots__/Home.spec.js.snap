// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot test renders correctly 1`] = `
<div>
  <h1>
    React-boilerplate
  </h1>
  <img
    alt="react-icon"
    src=""
  />
  <p>
    Welcome to simple react boilerplate.
  </p>
  <a
    href="https://github.com/Godsenal/react-boilerplate"
  >
    Github
  </a>
  <p>
    Example
  </p>
  <a
    href="/todo"
    onClick={[Function]}
  >
    TODO
  </a>
</div>
`;
